readme.md

```diff
@@ -41,16 +37,16 @@ $ npm install --save electron-debug
 var app = require('app');
 
 require('electron-debug')({
-	showDevTools: true
+  showDevTools: true
 });
 
 let win;
 
 app.on('ready', function () {
-	win = new BrowserWindow({
-		width: 800,
-		height: 600
-	});
+  win = new BrowserWindow({
+    width: 800,
+    height: 600
+  });
```

Comment:
I already use it, but I don't know why, on my linux PC sublime give precedence to my user preferences. On my MAC in office, it correctly does the opposite...